internal/app/upstream/multiplexer.go

```diff
@@ -0,0 +1,34 @@
+package multiplexer
+
+import (
+	"context"
+
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	"google.golang.org/grpc"
+)
+
+// Multiplexer handles the connections to the upstream control plane
```

Comment:
makes sense.done.

Reply:
Sorry to be even more picky. I think it's important we establish a consistent glossary. Matt chose to use the term `origin server` when setting up the repo and blog post, and I think it makes sense to go with that terminology: `Caching, aggregation, and relaying for xDS compliant clients and origin servers`

Reply:
That makes sense, 
Instead of `Multiplexer handles the connections to the upstream management server` how does this sound?
`Multiplexer handles the connections to the origin server`

Reply:
done